### ISSUE TITLE
re-arranged content with unbalanced fences

### ISSUE THREAD

**Issue by pkra** [Speech-Rule-Engine/speech-rule-engine#667]
Minimal example:

`{}= x\left. \operatorname {X} \left( \left( x \right)\right. \right. `

which ends up with the operator at the end of the line (when enriching):

![screenshot of rendered expression](https://user-images.githubusercontent.com/426686/180599573-b1a69dc2-fd86-4399-96c2-895f27158aed.png)

**Comment by pkra** [comment]
Original full example (which has unbalanced fences because of a manual linebreak)

```
\begin{aligned}
& u_2(t,x) \\ 
& {}= \frac{2}{t^{1/2}}\left. \operatorname {Re} \left( \left( \tilde{\Psi }(y) ( \log t + \log \sqrt {1-|y|^2}) +\tilde{\Phi }_2 (y) \right)\right. \right. \\ 
& \quad \left. \left. \times \exp \left(it\sqrt {1-|y|^2} +i\Psi (y) \log t \right) \right)\right|_{y=\frac{x}{t+2B}}\\ 
& \quad + O(t^{-\frac{3}{2}+c\delta }) \end{aligned}
```

**Comment by pkra** [comment]
Just for completeness: removing the extraneous `\left. ... \right.` (starting before the operator) seems to work around this issue.

**Comment by zorkow** [comment]
I can't reproduce that problem. Does that still exist?

**Comment by pkra** [comment]
This seems to have been fixed in v4.1.0-beta.0.
